grq2/services/api_v02/datasets.py

```diff
@@ -124,7 +155,7 @@ class IndexDataset(Resource):
     @grq_ns.expect(parser, validate=True)
     def post(self):
         # get bulk request timeout from config
-        bulk_request_timeout = app.config.get('BULK_REQUEST_TIMEOUT', 10)
+        bulk_request_timeout = app.config.get('BULK_REQUEST_TIMEOUT', 15)
```

Comment:
Just curious, do we know the reason of why we moved away from the default set in the config of 30?

https://github.com/hysds/grq2/blob/dcc4cb19b474a9b893b633ebfe0ea967b61dd68a/config/settings.cfg.tmpl#L18

Reply:
looks like the original default value is 10 seconds according to this PR: https://github.com/hysds/grq2/pull/57

Reply:
Oh my bad. It was too early in the morning when I reviewed this, but I interpreted this change as we were increasing the bulk request timeout from 10 to 15 when in reality it's actually 30 seconds since it reads from the config. So, whether this is 10 or 15 doesn't really matter I guess.